pkg/provisioning/provisioning.go

```diff
@@ -4,136 +4,35 @@ import (
 	"time"
```

Comment:
Would it make sense to rename this package to provisioner?

Also, instead of pkg/ironic/, how about pkg/provisioner/ironic/ ?

Reply:
Yeah, the rename makes sense.

I could move the directory, too. I wasn't sure whether we wanted flatter or deeper structure.

Reply:
Done